jekyll/_cci2/server-3-install-migration.adoc

```diff
@@ -46,10 +42,12 @@ The migration script will:
 * Stop your v2.19.x application
 * Create a tarball of your v2.19.x application's PostgreSQL and Mongo databases
 * Archive existing application data for Vault and CircleCI encryption/signing keys
-* Export the tarball to your v3.x installation (exported data stores are stored in `migrate/circleci_export`, which may be useful for debugging purposes).
+* Export the tarball to your v3.x installation (exported data stores are stored in `circleci_export`, which may be useful for debugging purposes).
 * Scale v3.x application deployments down to zero, and then to one
 * Import the data from the previously exported tarball to your new v3.x instance
 
+NOTE: If your services are externalized then you may run `./migrate.sh --skip-databases` to skip Postgres and Mongo. In the event Postgres, Mongo, and Vault are all externalized, you simply need to `scp` (secured copy) your keys from `/data/circle/circleci-encryption-keys` on the v2.19.x services machine.
```

Comment:
Where do I copy them too?

```suggestion
NOTE: If your services are externalized then you may run `./migrate.sh --skip-databases` to skip Postgres and Mongo. In the event Postgres, Mongo, and Vault are all externalized, you need to `scp` (secured copy) your keys from `/data/circle/circleci-encryption-keys` on the v2.19.x services machine.
```

Reply:
Where do I copy the keys to?

Reply:
Welp, I definitely fixed this but now I don't know where that commit went. Sorry. I will have to fix this tomorrow.

Reply:
Ah. I never `git push`d it. So embarrassing xD